srcpkgs/ydotool/files/ydotool-workaround

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+YDOTOOL_SOCKET="${YDOTOOL_SOCKET:-/var/lib/ydotoold/socket}" /usr/libexec/ydotool "$@"
```

Comment:
Are you doing this because you want to change the default socket path? Why? Can we have a patch instead?

Reply:
I've put a patch instead with an explanation. I also adjusted the the other patch as you requested above.